include/meson.build

```diff
@@ -0,0 +1,23 @@
+header_files = [
+    'vir/detail.h',
+    'vir/simd_bit.h',
+    'vir/simd_bitset.h',
+    'vir/simd_cast.h',
+    'vir/simd_float_ops.h',
+    'vir/simd_resize.h',
+    'vir/simd.h',
+    'buffer_skeleton.hpp',
+    'buffer.hpp',
+    'circular_buffer.hpp',
+    'claim_strategy.hpp',
+    'graph.hpp',
+    'node_traits.hpp',
+    'node.hpp',
+    'port_traits.hpp',
+    'port.hpp',
+    'sequence.hpp',
+    'typelist.hpp',
+    'utils.hpp',
+    'wait_strategy.hpp'
+]
+install_headers(header_files, subdir : 'graph-prototype')
```

Comment:
So e.g. `vir/detail.h` should be installed to the system as `/usr/include/graph-prototype/detail.h`? This seems wrong -- at least vir/ should keep its structure.

Meson 0.63.0 added support for that with `preserve_path: true`.